model/participants.go

```diff
@@ -25,6 +25,8 @@ var (
 	RoleRemixer     = Role{"remixer"}
 	RoleDJMixer     = Role{"djmixer"}
 	RolePerformer   = Role{"performer"}
+	// Credit where the artist is an album artist or artist
```

Comment:
Prefixing the comment with the name of the symbol makes it a godoc, used in hits by IDEs
```suggestion
	// RolePrimary is a credit where the artist is an album artist or artist
```